ci/configure/openshift.sh

```diff
@@ -9,8 +9,11 @@ BRIDGE_K8S_MODE="off-cluster"
 BRIDGE_K8S_MODE_OFF_CLUSTER_SKIP_VERIFY_TLS=true
 BRIDGE_USER_SETTINGS_LOCATION="localstorage"
 
-# If we have oc tool and an Openshift token, assume we are connected to openshift
-if oc whoami --show-token; then
+if [ -n "${BRIDGE_K8S_MODE_OFF_CLUSTER_ENDPOINT-}" ]; then
```

Comment:
@rszwajko hi, this renders the checks for `BRIDGE_K8S_MODE_OFF_CLUSTER_ENDPOINT` null 
I would prefer a separate env var ? `USE_FIXED_K8S_ENDPOINT` ? and test for it to exist , WDYT ?

Reply:
the checks in lines 19 and 22

Reply:
> this renders the checks for BRIDGE_K8S_MODE_OFF_CLUSTER_ENDPOINT null
>
right.  we can remove them and use direct assignment.

> I would prefer a separate env var ? USE_FIXED_K8S_ENDPOINT ? and test for it to exist , WDYT ?
>
IMHO they are OK as they are.  They are  part of the dev API provided by the Console. In my env I use fixed value for both `BRIDGE_K8S_MODE_OFF_CLUSTER_ENDPOINT` and  `BRIDGE_K8S_AUTH_BEARER_TOKEN`.